googleapiclient/http.py

```diff
@@ -24,34 +24,35 @@
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
-import StringIO
+from six import BytesIO, StringIO
+from six.moves.urllib.parse import urlparse, urlunparse, quote, unquote
```

Comment:
I think I found what I was looking for.  I overlooked it because I thought it was more in depth.

https://code.google.com/p/google-api-python-client/wiki/BecomingAContributor

    Submitting Patches
    ...
    3. Ensure that your code adheres to the PEP 8 style, with the exception of spacing, which should be two spaces.
    ...